src/lib.c

```diff
@@ -1,5 +1,5 @@
 #include <stdio.h>
 
 void hello_c() {
-	puts("Hello from C!");
+    puts("Hello from C!");
```

Comment:
Tabs are variable width and can be set to anything while spaces are fixed width. I'm not sure if this change was intentional, but I don't like it.